static/app/components/events/autofix/autofixChanges.tsx

```diff
@@ -419,8 +176,26 @@ export function AutofixChanges({
   isChangesFirstAppearance,
 }: AutofixChangesProps) {
   const {data} = useAutofixData({groupId});
-  const [isBusy, setIsBusy] = useState(false);
+  const isBusy = step.status === AutofixStatus.PROCESSING;
   const iconCodeRef = useRef<HTMLDivElement>(null);
+  const [isPrProcessing, setIsPrProcessing] = useState(false);
+  const [isBranchProcessing, setIsBranchProcessing] = useState(false);
```

Comment:
could we make these conditions completely reliant on the autofix state instead of being react state? We can just rely on the changes step being in processing status right? Will make all this logic much simpler

If on the seer side we update the status before we return the call then the query invalidation inside `onSuccess` should already receive a processing state next time it makes a request